tests/test_keycloak_setup.py

```diff
@@ -994,6 +994,10 @@ def test_create_client_exists(self):
             responses.GET, kc_clients_uuid_url, status=200, json=[{'id': "12345"}],
             headers={'location': str(mock.sentinel.location)})
 
+        kc_client_url = kc_clients_url + "/12345"
+        responses.add(
+            responses.PUT, kc_client_url, status=204)
+
         client.create()
 
         self.assertEqual(2, len(responses.calls))
```

Comment:
It looks like you added a response, so this check needs to be changed from `2` to `3`?

Reply:
Thanks